src/guides/v2.3/frontend-dev-guide/layouts/layout-create.md

```diff
@@ -0,0 +1,36 @@
+---
+group: frontend-developer-guide
+title: Create a new layout
+functional_areas:
+  - Frontend
+---
+
+### Create a new page layout in custom theme
+
+If an existing page layout doesn't meet your requirements, then you can create a new page layout in Magento.
```

Comment:
Try to avoid contractions. They are harder to translate. "doesn't" -> "does not".

Reply:
@dobooth Commit the changes and I will that advice in mind in my next PR. 
Thanks for the suggestion.